### ISSUE TITLE
Selecting multiple regions in region list for deletion

### ISSUE THREAD

**Issue by Kechil** [CARTAvis/carta-frontend#1040]
Usability Review:
I cannot clear all regions (or could not find how to do it). Could not select multiple regions on the list at once (I tried shift clicking but this did nothing. This behaviour is desirable if I wanted to delete a whole bunch of regions at once).

Deleting regions needs careful consideration.

**Comment by keflavich** [comment]
I second this.  I also really want a way to delete all regions so I can open a new region file.

**Comment by kswang1029** [comment]
refer to #1955

**Comment by keflavich** [comment]
Is there a frontend javascript/typescript approach that would let us do this now?

**Comment by veggiesaurus** [comment]
> Is there a frontend javascript/typescript approach that would let us do this now?

v3 only supports single-selected regions, so I think you'd have to loop through the list of regions in the code snippet and delete them. Is that what you meant @keflavich ?

**Comment by keflavich** [comment]
Yes, that's what I mean.  I tried again to dig through the source, but I didn't manage to find the container of regions to loop through.

**Comment by keflavich** [comment]
I've figured out how to do this now:
```js
for (const reg of app.frames[0].regionSet.regions) {
    app.frames[0].regionSet.deleteRegion(reg);
}
```
